schema.graphql

```diff
@@ -485,11 +544,78 @@ type MatchQuestionBuzz {
 """
 Docs group: Match
 """
-type MatchQuestionBonus {
+type BuzzPosition {
+  """
+  The time of the buzz. An ISO 8601-formatted string, with time zone, in the tournament's local time zone. May not be before the start time of reading the relevant question.
+  """
+  time: Date
+
+  """
+  The (`0`-based) index of the word on which the player buzzed.
```

Comment:
Maybe "The index of the word on which the player buzzed, where `0` is any time during the first word"?

Reply:
I'll give more explicit instructions.